extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -306,7 +306,8 @@ public static function onImageBeforeProduceHTML(
 			$details['context'] = self::isIconImage( $details, $handlerParams ) ? self::MEDIA_CONTEXT_ICON :
 				self::MEDIA_CONTEXT_ARTICLE_IMAGE;
 
-			$media = self::createMediaObject( $details, $title->getText(), $frameParams['caption'], $linkHref );
+			$caption = isset( $frameParams['caption'] ) ? $frameParams['caption'] : null;
```

Comment:
?? checks if variable is null, not sure how this will perform on missing key in array